samcli/lib/utils/osutils.py

```diff
@@ -53,7 +50,11 @@ def mkdir_temp(mode=0o755, ignore_errors=False):
 
 
 def _rmtree_callback(function, path, excinfo):
```

Comment:
I would recommend adding a docstring here, not as detailed as public methods, but a minor one explaining that this method will modify folder permission.

Reply:
One more question, do you want to prefix this method name with `_` given that this method is imported in another module?

Reply:
Done! 👏